### ISSUE TITLE
Dev: Runtime error from preferences - null species

### ISSUE THREAD

**Issue by skull132** [Aurorastation/Aurora.3#1361]
It seems that a preferences datum cannot be trusted to always have a species? That, or the grabber @LordFowl wrote is unreliable.

```
runtime error: 
[23:31:02]Cannot read null.age_min
[23:31:02]proc name: getMinAge (/datum/preferences/proc/getMinAge)
[23:31:02]  source file: preferences.dm,153
runtime error: 
[23:31:02]Cannot read null.age_max
[23:31:02]proc name: getMaxAge (/datum/preferences/proc/getMaxAge)
[23:31:02]  source file: preferences.dm,157
```

```
runtime error: 
[23:33:17]Cannot execute null.get bodytype().
[23:33:17]proc name: OnTopic (/datum/category_item/player_setup_item/general/body/OnTopic)
[23:33:17]  source file: 03_body.dm,317
```

**Comment by Lohikar** [comment]
Probably connected to #1358.

**Comment by Lohikar** [comment]
Should be fixed by #1370.
